ivy/func_wrapper.py

```diff
@@ -369,6 +370,12 @@ def new_fn(*args, out=None, **kwargs):
 
 def handle_nestable(fn: Callable) -> Callable:
     fn_name = fn.__name__
+    try:
```

Comment:
The static container functions must be defined at import time, the methods are public and it should be possible to call them in isolation of the nestable behaviour of the functions in the functional API. It should be possible to solve this at import time, by re-arranging some things, but it's not an easy feat. Let me know if you have any questions!

Reply:
Understood, will do. Thanks.